src/aspire/commands/preprocess.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import click
+
+from aspire.estimation.noise import WhiteNoiseEstimator
+from aspire.source.relion import RelionSource
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option('--data_folder', default=None,
+              help='Path to data folder')
+@click.option('--starfile_in', required=True,
+              help='Path to input starfile relative to data folder')
```

Comment:
Throughout:
“starfile” → “STAR file“
“mrc file” → “MRC file”

Reply:
Done in new commit 65857e4.